catalog-rest-service/src/main/resources/ui/src/components/my-data-details/SchemaTable.test.tsx

```diff
@@ -19,14 +19,14 @@ import { getAllByTestId, render } from '@testing-library/react';
 import { TableDetail } from 'Models';
 import React from 'react';
 import { MemoryRouter } from 'react-router-dom';
-import { Column, ColumnDataType } from '../../generated/entity/data/table';
+import { ColumnDataType } from '../../generated/entity/data/table';
 import SchemaTable from './SchemaTable';
 
 jest.mock('../common/rich-text-editor/RichTextEditorPreviewer', () => {
   return jest.fn().mockReturnValue(<p>RichTextEditorPreviewer</p>);
 });
 
-const mockColumns: Column[] = [
```

Comment:
Why did we remove the Column type?

Reply:
its work completely fine without it so in test suit we only add things if it is required.

Reply:
![image](https://user-images.githubusercontent.com/71748675/134757593-93f84f7b-d92b-4f9b-8273-da9bcc485a0b.png)

Reply:
@ShaileshParmar11  Have you checked typescript errors as well?
when you locally run the application please check if it is working fine or not.

Reply:
I removed it from the test file and not the original component, and everything is working fine, without any new warning/error.